sunpy/net/hek/tests/test_utils.py

```diff
@@ -0,0 +1,117 @@
+import json
+
+import numpy as np
+import pytest
+from regions import PolygonSkyRegion
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.net import attrs, hek
+from sunpy.net.hek.tests.test_hek import HEKResult
+from sunpy.net.hek.utils import COORD_FILE_PATH, UNIT_FILE_PATH, get_unit, parse_chaincode
+
+_hek_result = HEKResult()
+
+@pytest.fixture
+def read_unit_attributes():
+    with open(UNIT_FILE_PATH) as unit_file:
+        unit_properties = json.load(unit_file)
+
+    return unit_properties
+
+@pytest.fixture
+def read_coord_attributes():
+    with open(COORD_FILE_PATH) as coord_file:
+        coord_properties = json.load(coord_file)
+
+    return coord_properties
+
+@pytest.mark.remote_data
+def test_missing_times():
+    # Check for https://github.com/sunpy/sunpy/pull/7627#issuecomment-2113451964
+    client = hek.HEKClient()
+    results = client.search(attrs.Time('2024-05-10', '2024-05-12'), attrs.hek.AR.NOAANum == 13664)
+    assert isinstance(results["event_peaktime"][0], np.ma.core.MaskedConstant)
+    assert results["event_peaktime"][6].isot == "2024-05-10T16:08:00.000"
+
+@pytest.mark.remote_data
+def test_astropy_unit_parsing(read_unit_attributes, read_coord_attributes):
```

Comment:
These unit tests check for the return from the client search. These belong in the hek file.

Reply:
Ok